### PR TITLE
Create Logging.md

### PR DESCRIPTION
**Pull request by janiskelemen**
Initial documentation for the new tenant aware logger.

### CODE REVIEW

**Comment by luceos** [comment]
relates to https://github.com/hyn/multi-tenant/pull/524

File: docs/hyn/5.2/Logging.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Logging
+icon: fal fa-book
+---
+
+Especially in production you might want to split the default laravel log by tenants.
+Tenancy shipps with a custom tenant aware logger which you can register in the logger config.
```
Comment (fletch3555) [inline docs/hyn/5.2/Logging.md]: `shipps` should be `ships`

File: docs/hyn/5.2/Logging.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+If a valid hostname is detected it will create a `logs` folder within the tenant directory ie.:
+`app/tenancy/tenants/xxxxxxxxxxxxxxxxxxxx/logs/{log_level}_xxxx-xx-xx.log`
```
Comment (fletch3555) [inline docs/hyn/5.2/Logging.md]: Could you clarify where `app/....` is rooted within the project? The next example starts with `/storage...`, which clearly identifies it as a path starting from the project root.  `app/...` does not provide such context.

File: docs/hyn/5.2/Logging.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+If no valid hostname is detected it will default to:
+`/storage/logs/{log_level}_xxxx-xx-xx.log`
+
+Add a new `tenant` channel to the channels array in your **config/logger.php** like below:
+
+```php
+...
```
Comment (fletch3555) [inline docs/hyn/5.2/Logging.md]: I don't think the `...` are necessary.  Someone may think they are, copy/paste them, then complain it's not working.  One would need to be very novice to do so, but it's entirely plausible.

File: docs/hyn/5.2/Logging.md (hunk 2 above)
Comment (janiskelemen) [reply docs/hyn/5.2/Logging.md]: Yes right, should be /storage instead of /app.

File: docs/hyn/5.2/Logging.md (hunk 3 above)
Comment (janiskelemen) [reply docs/hyn/5.2/Logging.md]: Agreed.

**Pull request merged**